src/lib/nvme_pt_ioctl.c

```diff
@@ -825,6 +829,37 @@ static int opal_revert_tper_local(int fd, struct opal_device *dev)
 	return ret;
 }
 
+static struct opal_req_item opal_revert_sp_cmd[] = {
+	{.type = OPAL_U8, .len = 1, .val = {.byte = OPAL_STARTNAME}},
+	{.type = OPAL_U8, .len = 1,
+		.val = {.byte = opal_mehod_params[OPAL_REVERTSP_KEEP_GL_RN_KEY][0]}},
+	{.type = OPAL_U8, .len = 1,
+		.val = {.byte = opal_mehod_params[OPAL_REVERTSP_KEEP_GL_RN_KEY][1]}},
+	{.type = OPAL_U8, .len = 1,
+		.val = {.byte = opal_mehod_params[OPAL_REVERTSP_KEEP_GL_RN_KEY][2]}},
+	{.type = OPAL_U8, .len = 1,
+		.val = {.byte = opal_mehod_params[OPAL_REVERTSP_KEEP_GL_RN_KEY][3]}},
+	{.type = OPAL_U8, .len = 1,
+		.val = {.byte = OPAL_TRUE}},
+	{.type = OPAL_U8, .len = 1,
+		.val = {.byte = OPAL_ENDNAME}},
+};
+
+static int opal_revert_sp_local(int fd, struct opal_device *dev, bool keep_global_rn_key) {
+    int ret = 0;
+
+    prepare_req_buf(dev, opal_revert_sp_cmd, ARRAY_SIZE(opal_revert_sp_cmd),
```

Comment:
please conform to the coding standard: use tabs for indentation instead of spaces

Reply:
done